content/telegraf/v1.17/about_the_project/release-notes-changelog.md

```diff
@@ -7,6 +7,31 @@ menu:
     weight: 10
     parent: About the project
 ---
+## v.1.17.3 [2021-2-17]
+
+### Input plugin updates
+
+- Filestat (`filestat`): Skip missing files.
+- x509 Certificate (`x509_cert`): Fix a timeout issue.
+- MQTT Consumer (`mqtt_consumer`): Fix reconnection issues.
+- SNMP (`snmp`): Add support to expose IPv4/IPv6 as connection-schemes.
+- Ping (`ping`):
+        - Fix a timeout for `deadline` configuration.
+        - Update README with correct cmd for native ping on Linux.
+        - Fix percentile calculations.
+### Output plugin updates
+
+- Warp10 (`warp10`): url encode comma in tags value.
+- InfluxDB v1.x (`influxdb`): Validate InfluxDB response after creating a database to avoid JSON parsing errors.
+
+### Miscellaneous fixes and updates
+
+- Update to Go 1.15.8.
```

Comment:
Tim had commented he wanted the Go version updates at the top - so people know the latest version of Go Telegraf is running

Reply:
Good to know, thanks Sam. Moved that item up! 

Tim had mentioned we should put some software version upgrades in a "maintenance/fixes section" so good one to get consistent across products. 

Background: In this Chronograf [PR](https://github.com/influxdata/docs-v2/pull/2142/files), Tim requested we move 'Upgrade to Axios 0.21.1' to the bug fixes section.